helm/happa/templates/ingress.yaml

```diff
@@ -30,8 +30,4 @@ spec:
   tls:
   - hosts:
     - {{ .Values.Installation.V1.GiantSwarm.Happa.Host }}
-    {{- if .Values.Installation.V1.GiantSwarm.Happa.Letsencrypt }}
-    secretName: giantswarm-happa-tls
-    {{- else }}
-    secretName: happa-certs-secret
-    {{- end }}
+    secretName: happa-certs
```

Comment:
Using different secret name looked strange to me. I put if around the secret template instead.